README.md

```diff
@@ -1,46 +1,101 @@
-* `PokeminiD` - for debugging the compiled .min images
-* `pokeflash` - for flashing the images into a flash cart
+* `PokeMiniD` - for debugging the compiled .min images
 
-With all of that installed and in the path you can then
+```sh
+# On Ubuntu and other Debian-based distros
+sudo apt install make git curl wget unzip unshield srecord
+# ...for 32 bit versions:
+sudo apt install wine
+# ...for 64 bit versions:
+sudo apt purge wine
+sudo dpkg --add-architecture i386
+wget -qO - https://dl.winehq.org/wine-builds/winehq.key | sudo apt-key add -
+sudo apt-add-repository "deb https://dl.winehq.org/wine-builds/ubuntu/ $(lsb_release -sc) main"
+sudo apt update && sudo apt install --install-recommends winehq-stable
 
-``` bash
-$ git clone "https://github.com/pokemon-mini/c88-pokemini"
-$ cd c88-pokemini
-$ make
+# Install pokemini dev (make sure to have $HOME/.local/bin in PATH)
+curl -Lo pokemini_060_linux32dev.zip https://sourceforge.net/projects/pokemini/files/0.60/pokemini_060_linux32dev.zip/download
+unzip pokemini_060_linux32dev.zip PokeMiniD -d ~/.local/bin
+
+# Install these tools
+git clone "https://github.com/pokemon-mini/c88-pokemini"
+cd c88-pokemini
+make
 ```
 
-## Windows
+### Windows
+
+Windows is supported through MSYS2, just install the base system from https://www.msys2.org/ You then can use the msys2 shell to clone this repository and build the toolchain.
 
-Windows is supported through MSYS2, just install the base system from
-https://www.msys2.org/ You then can use the msys2 shell to clone this
-repository and build the toolchain.
+Some tools are downloaded automatically by the setup script: unshield, srec_cat, PokeMiniD. You must place the PokeMiniD location on your PATH in order to use `mk88 run` in projects, whether it's your own version or the included one.
 
-``` bash
+```sh
 # Ensure dependencies
-$ pacman -S git unzip curl make
+pacman -S git unzip curl make
 # From here it goes as before
-$ git clone "https://github.com/pokemon-mini/c88-pokemini"
-$ cd c88-pokemini
-$ make # The makefile will automatically download dependencies and build the toolchain folder
+git clone "https://github.com/pokemon-mini/c88-pokemini"
+cd c88-pokemini
+make # The makefile will automatically download dependencies and build the toolchain folder
+```
+
+### OSX
+
+OSX no longer supports running 32 bit binaries, so running the suite on the bare OS is not possible. You will have to run them in a VM.
+
+## Building the example
+
+You may use CMD, MSYS, or a Linux shell emulator. It's recommended to have c88tools/bin in the PATH but not required.
+
+### Commands
+
+The makefile comes with the following commands:
+
+* `all` - the default, build the min
+* `clean` - clean up all the build object
+* `assembly` - build all C files to their assembler forms
+* `run` - build and run the min in PokeMiniD
+* `src/main.src` - build main.c to its assembler form (can do this with any C file)
+
+### Linux
+
+```sh
+# Start in the example's directory
```

Comment:
```suggestion
# Start in the examples directory
```

Reply:
This isn't correct. If you want it to be the name of the directory, it would be "helloworld" but it says that right below, so I'm not sure that's necessary.